tests/test_data/cropA/pyrate_mexico_cropa.conf

```diff
@@ -15,6 +15,9 @@ apsest:        1
 # DEM error (residual topography) correction (CORRECT)
 demerror:      1
 
+# Phase Closure correction (CORRECT)
+phase_closure:      1
+
```

Comment:
Please add to the top level `input_parameters.conf` - this config file is more visible to users

Reply:
addressed by https://github.com/GeoscienceAustralia/PyRate/pull/307/commits/9f840bc6d9ffd818c5e7cf6c99a91be12696403a